extensions/amp-accordion/1.0/accordion.js

```diff
@@ -189,6 +193,7 @@ export function AccordionSection({
 
   const expanded = isExpanded ? isExpanded(id, defaultExpanded) : expandedState;
   const animate = contextAnimate ?? defaultAnimate;
+  const contentId = prefix + '_AMP_content_' + id;
```

Comment:
Couple of things here:
1. `prefix` might not be available here since contexts are always optional. So we need to reflect it here.
2. Please use backticks.

E.g. this might end up looking like this:

```
const contentId = `${prefix || 'a'}-content-${id}`
```

Reply:
Why not just use the random ID generator for `const [id] = useState(...)` above?

Reply:
I actually like that the `id` from `const [id] = useState(...)` is sequential, because it conveys some information to the user which section we are on (it's like an index).  If you feel strongly about it, I can update to your recommendation.

Reply:
It can also lead to some incorrect expectations, since we do not otherwise guarantee any kind of ID ordering :)

Reply:
Ahh that's good to keep in mind.  Good point!